var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -202,6 +202,12 @@ def flag_handler(self, name, flags):
             if self.spec.satisfies('@:1.8.12+cxx~shared'):
                 cmake_flags.append(self.compiler.cxx_pic_flag)
         elif name == "fflags":
+            if self.spec.satisfies('%cce+fortran'):
+                # Cray compiler generates module files with uppercase names by
+                # default, which is not handled by the CMake scripts. The
+                # following flag forces the compiler to produce module files
+                # with lowercase names.
```

Comment:
Is there an upstream bug report that we could reference here?
https://github.com/HDFGroup/hdf5/issues